cli.js

```diff
@@ -152,6 +153,15 @@ function populateDB(postgresConnectionString) {
              4. dbpedia works/releases (as array in the artists input)
              */
 
+            //prepopulate instrumentsArray
+            const instruments = context.models.instruments;
+            instruments.findAll().then((queriedInstruments)=>{
```

Comment:
This will become slow if the amount of instruments is real high. Though, it _works_, for now